jetty/install_windows_src.md

```diff
@@ -13,23 +13,19 @@ without any failures when using their functionalities.
 > You should be able to use `ogre` as a rendering engine instead of the default `ogre2`.
 > Just append `--render-engine ogre` to the command line.
 
+## Dependency distribution (conda-forge) and the package manager (Pixi)
+
+Binaries for all the dependendencies used by Gazebo can be found in the [conda-forge](https://conda-forge.org/)
+package repository. The Gazebo buildfarm and these instructions uses the [Pixi](https://pixi.sh/) package manager
+but other package managers for Conda should work like miniforge, mamba, etc.
+
 ## Install dependencies
 
-1. If you do not have the conda package manager installed in your system, install a conda distribution.
-   As Gazebo uses all dependencies from the conda-forge channel, we suggest to install miniforge following [the official miniforge installation docs](https://github.com/conda-forge/miniforge#windows) by executing the following commands in a "Command Prompt" terminal:
-   ```cmd
-   curl.exe -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
-   start /wait "" Miniforge3-Windows-x86_64.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%UserProfile%\Miniforge3
-   %UserProfile%\Miniforge3\condabin\conda init
-   %UserProfile%\Miniforge3\condabin\conda config --set auto_activate_base false
-   del Miniforge3-Windows-x86_64.exe
-   ```
-   If you followed this step correctly, you should be able to access `conda` in any new "Command Prompt" terminal you open.
-   To verify that, check that if you open a new terminal and execute `conda info`, the output begins with:
-   ```cmd
-     active environment : None
-            shell level : 0
-     [..]
+1. If no package manager is installed in the system to manage conda-forge dependencies, the recommended
+   option is to follow the Pixi installer instructions.
```

Comment:
It's now perhaps not clear that this needs to be executed in the command prompt as it was before.